### ISSUE TITLE
Does Assimp now support to export the Colload for V1.5.0 ? I am now using the Assimp.Net to transfer the obj to dae file.

### ISSUE THREAD

**Issue by ZhangDonglin12** [assimp/assimp#3713]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by kimkulling** [comment]
I just checked this. at the moment only collada version 1.4.1 will be exported. Do you need the newer version?

**Comment by ZhangDonglin12** [comment]
yeah, Do you have the  plan to upgrate to collada version1.5? Thank you so much!

**Comment by ZhangDonglin12** [comment]
> I just checked this. at the moment only collada version 1.4.1 will be exported. Do you need the newer version?

yeah, Do you have the plan to upgrate to collada version1.5? Thank you so much!  what about the difference for Colloada-dom and in the Assimpt to generate colloada model file?

 I am now using model file to do collision check in Simulation, The collision checker can only deal with convex models.
 So I need to use V-HACD to split to convex models to multiple convex model file. (https://github.com/kmammou/v-hacd) . 
 Do you have any other advice for me to do this feature? Thank you so much!

**Comment by XXxR4GALXxx** [comment]
I also need collada 1.5. Exporter 1.4, is incomplete (does not work correctly with multiple animations). Other than that, most 3D editors work with collada 1.5.

**Comment by XXxR4GALXxx** [comment]
i so need collada 1.5+. (collada 1.4 is very old like 2006)

**Comment by kimkulling** [comment]
OK, then we need to add the 1.5-support to the Collada-exporter. Maybe you could help us here. At the moment I don't see so much free time to do this job I have to say.

**Comment by tellypresence** [comment]
Please add "Export" label to this issue

**Comment by tellypresence** [comment]
Mentioned in epic #6097, if someone picks this up please reopen

**Comment by turol** [comment]
"help wanted" tag is for issues where the project wants help from the larger Github community members. It will show up in some automated lists. I don't think qualifies. The 3D formats megaissue probably does though.
